rules/splice.bzl

```diff
@@ -61,19 +61,25 @@ def _bitstream_splice_impl(ctx):
         env = ENV,
     )
 
-    ctx.actions.run(
-        mnemonic = "UpdateUsrAccessValue",
-        outputs = [output],
-        inputs = [spliced],
-        arguments = [
-            "--logging",
-            "info",
-            "update-usr-access",
-            spliced.path,
-            output.path,
-        ],
-        executable = ctx.executable._opentitantool,
-    )
+    if ctx.attr.update_usr_access:
+        ctx.actions.run(
+            mnemonic = "UpdateUsrAccessValue",
+            outputs = [output],
+            inputs = [spliced],
+            arguments = [
+                "--logging",
```

Comment:
Add `--rcfile=` at the beginning of the argument list to prevent opentitantool from reading the user's `$HOME/.config/opentitantool/config`.

Reply:
Thanks for catching this! Done.